kohese-portal/client/src/components/tree/default-tree/default-tree.component.spec.ts

```diff
@@ -135,24 +137,21 @@ describe('Component: default-tree', () => {
     }
     expect(index).toEqual(-1);
 
-    TestBed.get(ActivatedRoute).params.next({ id: 'Item' });
-    tick();
-
     for (let j: number = 0; j < component.visibleRows.length; j++) {
-      if ('Item' === (component.visibleRows[j].object as ItemProxy).item.id) {
+      if ('Model-Definitions' === (component.visibleRows[j].object as ItemProxy).item.id) {
```

Comment:
What is the purpose of this change?

Reply:
This has been resolved in angular-upgrade-12

document as:
Issue: default-tree component test synchronizes with selection does nothing (d122ec00-051d-11ec-9403-11f290226b38)